File: Password.py
number=input("Enter your password: ")

numbers = len(number)

if numbers  > 6 and numbers <= 10:
    print("Medium")
if( numbers < 6 ):
    print("Weak")
if( numbers > 10):
    print("Strong")
if(numbers  < 1):
     print("is invalid")

File: Password.java
import java.util.Scanner;
public class Password{
public static void main(String[]args){
Scanner input=new Scanner(System.in);

System.out.print("Enter your password: ");
String password=input.nextLine();

int length=password.length();

if (length  > 6 && length <= 10){
 System.out.println("Medium");
}
if( length < 6 ){
System.out. println("Weak");
}
if( length > 10){
System.out.println("Strong");
}
if(length < 1){
 System.out.println("is invalid");
}

}
}

